mava/wrappers/jaxmarl.py

```diff
@@ -226,3 +262,16 @@ def discount_spec(self) -> specs.BoundedArray:
         return specs.BoundedArray(
             shape=(self.num_agents,), dtype=float, minimum=0.0, maximum=1.0, name="discount"
         )
+
+    def action_mask(self, state: JaxMarlState) -> Array:
+        """Get action mask for each agent."""
+        if self.has_action_mask:
+            avail_actions = self._env.get_avail_actions(state)
+            mask = jnp.array(batchify(avail_actions, self.agents), dtype=jnp.float32)
+        else:
+            mask = jnp.ones(self._action_shape)
```

Comment:
```suggestion
            mask = jnp.ones(self._action_shape, dtype=jnp.float32)
```
Just to be explicit + consistent with the line above :)